src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LoggerMessageAttribute.cs

```diff
@@ -37,6 +37,20 @@ public sealed class LoggerMessageAttribute : Attribute
         /// </summary>
         public LoggerMessageAttribute() { }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LoggerMessageAttribute"/> class
+        /// which is used to guide the production of a strongly-typed logging method.
+        /// </summary>
+        /// <param name="eventId">The log event Id.</param>
+        /// <param name="level">The log level.</param>
+        /// <param name="message">Format string of the log message.</param>
+        public LoggerMessageAttribute(int eventId, LogLevel level, string message)
```

Comment:
Any reason not to have EventName here?

Reply:
This was the only ctor approved since we assumed it to be the most commonly used.